torchvision/io/gpu_decoder.py

```diff
@@ -0,0 +1,22 @@
+import torch
+
+
+class GPUDecoder:
+    def __init__(self, src_file: str):
+        torch.ops.load_library('build/lib.linux-x86_64-3.8/torchvision/Decoder.so')
```

Comment:
is it possible to register it through tochbind?

Reply:
What do you mean? Here, the library that was built needs to be loaded before they could be used. Similar to https://github.com/pytorch/vision/blob/ba299e8f52737448f1cbb9fffd25bf3d6e7ceef7/torchvision/io/image.py#L10.